src/web/pages/alerts/emailmethodpart.jsx

```diff
@@ -34,14 +34,13 @@ import Divider from 'web/components/layout/divider';
 import IconDivider from 'web/components/layout/icondivider';
 import Layout from 'web/components/layout/layout';
 
-import compose from 'web/utils/compose';
+import useCapabilities from "web/hooks/useCapabilities.js";
```

Comment:
One small detail. You can drop the `.js` suffix
```suggestion
import useCapabilities from 'web/hooks/useCapabilities';
```